tools/tox_tools/dose_responses/dose_response.xml

```diff
@@ -1,7 +1,7 @@
 <tool id="dr_curve" name="Dose Response Curve for Toxicological Risk Assessment" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.0">
     <description>for Toxicological Risk Assessment</description>
     <macros>
-        <token name="@TOOL_VERSION@">3.0.1</token>
+        <token name="@TOOL_VERSION@">4.0.1</token>
         <token name="@VERSION_SUFFIX@">3</token>
```

Comment:
```suggestion
        <token name="@TOOL_VERSION@">3.0.1</token>
        <token name="@VERSION_SUFFIX@">2</token>
```

Otherwise the version of requirement the r-drc requirement changes as well (and 4.0.1 does not exist yet).

If it still fails with this change then we might need to switch the version to 3.0_1, since 3.0.1 does [not exist](https://anaconda.org/bioconda/r-drc/files) .. but lets try first.